docs/sources/next/javascript-api/k6-experimental/websockets/_index.md

```diff
@@ -16,31 +15,32 @@ A global event loop lets a single VU have multiple concurrent connections, which
+| WebSocket.readyState                                                                                                                    | The current state of the connection. Could be one of [the four states](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/readyState).                                                                                                                                                                                                                                                                                                                            |
+| WebSocket.url                                                                                                                           | The URL of the connection as resolved by the constructor.                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| WebSocket.bufferedAmount                                                                                                                | The number of bytes of data that have been queued using calls to `send()` but not yet transmitted to the network.                                                                                                                                                                                                                                                                                                                                                         |
+| WebSocket.binaryType                                                                                                                    | **Required**. The [`binaryType`](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/binaryType) controls the type of binary data being received over the WebSocket connection. Use `"blob"` to receive it as `Blob`, or `"arraybuffer"` to receive it as `ArrayBuffer`. Currently, there is no default value, but in the future it will move to default to `"blob"`. If you want to have the same behaviour - you should always be setting it to `"arraybuffer"`. |
```

Comment:
```suggestion
| WebSocket.binaryType                                                                                                                    | _Required_. The [`binaryType`](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/binaryType) controls the type of binary data being received over the WebSocket connection. Use `"blob"` to receive it as `Blob`, or `"arraybuffer"` to receive it as `ArrayBuffer`. The default value is empty. That will change to default to `"blob"` in the future. If you want to keep the same behavior, we recommend setting it to `"arraybuffer"`. |
```

@joanlopez is it a 100% certainty that we're changing the default value to `blob`? Generally, we try to avoid making any statements in the docs about future behavior, because they can change and ideally we'd like to only document existing behavior, but I think this might be an exception.

Reply:
Yes, because it is what's defined on the spec (https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/binaryType).

However, as initially we only had support for `"arrayBuffer"`, we set that as the default behavior. Although, there was an alert warning users about future changes. Now, to avoid going from one to the other by default, we transitioned to making the attribute required, but the end goal is to be spec-compliant (so `"blob"` by default).

cc/ @mstoykov

Reply:
just a nitpick on the wording - it can't have a default value if it is required for you to set it. So maybe drop `The default value is empty` part.